front/app/containers/PasswordChange/index.tsx

```diff
@@ -133,6 +135,13 @@ const ChangePassword = ({ tenant }: Props) => {
         />
         <main>
           <StyledContentContainer>
+            <Box mt="30px">
+              <GoBackButton
+                onClick={() => {
+                  clHistory.goBack();
+                }}
+              />
+            </Box>
```

Comment:
Just used the opportunity to add a goBack button to the password change page as well, since it was missed.